src/post-message.ts

```diff
@@ -18,6 +18,14 @@ type RequestsFromNativeApp = {
     };
 };
 
+export type SheetResponse = {
+    action: 'SUBMIT' | 'DISMISS';
+    result: Array<{
+        id: string;
+        selected: Array<string>;
```

Comment:
not sure if too late but this `selected` should be `selectedIds` (like in the request message)

I know that the spec documents this as "selected"